### DISCUSSION TITLE
Auto Cooling

### DISCUSSION THREAD

**Discussion by DLovejoy1947** [arkypita/LaserGRBL#1122] [Q&A]
I have changed the configuration on laserGRBL to turn on cooling for 10 minutes of engraving. When I start engraving it does not stop after 60 minutes. If I click the red stop button the laser stops moving, the laser remains at full power.

I uninstalled laserGRBL v3.7.1 and downloaded a new copy. I installed and made the configuration changes, but no change to dooling.

Any ideas or help would be very welcome.

**Comment by BeatSlayer** [comment]
You most enable laser mode setting $32=1
